rpcs3/Emu/RSX/VK/VKOverlays.cpp

```diff
@@ -887,50 +887,80 @@ namespace vk
 			"}\n";
 
 		fs_src =
-			"#version 420\n\n"
-			"layout(set=0, binding=1) uniform sampler2D fs0;\n"
-			"layout(set=0, binding=2) uniform sampler2D fs1;\n"
-			"layout(location=0) in vec2 tc0;\n"
-			"layout(location=0) out vec4 ocol;\n"
-			"\n"
-			"layout(push_constant) uniform static_data\n"
-			"{\n"
-			"	float gamma;\n"
-			"	int limit_range;\n"
-			"	int stereo;\n"
-			"	int stereo_image_count;\n"
-			"};\n"
-			"\n"
-			"vec4 read_source()\n"
-			"{\n"
-			"	if (stereo == 0) return texture(fs0, tc0);\n"
-			"\n"
-			"	vec4 left, right;\n"
-			"	if (stereo_image_count == 2)\n"
-			"	{\n"
-			"		left = texture(fs0, tc0);\n"
-			"		right = texture(fs1, tc0);\n"
-			"	}\n"
-			"	else\n"
-			"	{\n"
-			"		vec2 coord_left = tc0 * vec2(1.f, 0.4898f);\n"
-			"		vec2 coord_right = coord_left + vec2(0.f, 0.510204f);\n"
-			"		left = texture(fs0, coord_left);\n"
-			"		right = texture(fs0, coord_right);\n"
-			"	}\n"
-			"\n"
-			"	return vec4(left.r, right.g, right.b, 1.);\n"
-			"}\n"
-			"\n"
-			"void main()\n"
-			"{\n"
-			"	vec4 color = read_source();\n"
-			"	color.rgb = pow(color.rgb, vec3(gamma));\n"
-			"	if (limit_range > 0)\n"
-			"		ocol = ((color * 220.) + 16.) / 255.;\n"
-			"	else\n"
-			"		ocol = color;\n"
-			"}\n";
+      "#version 420\n\n"
```

Comment:
The indentation in this whole section is altered and adds a lot of unnecessary changes to this PR.

Reply:
indentation issues are mostly due to vscode having a fun time, i'll be sure to fix them though.